bears/python/ShebangBear.py

```diff
@@ -0,0 +1,34 @@
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+            Ensure that the file begins with the Shebang operator.
+        """
+        if len(file) == 0 or '#!/usr/bin/env' in file[0]:
+            return
+
+        diff = Diff(file)
+        mes = ''
+
+        if '#!' in file[0]:
+            diff.modify_line(1, '#!/usr/bin/env')
```

Comment:
@AMR-KELEG It's ok. Please don't wait for our response just do the changes needed ASAP. We're pretty busy with a lot of stuff and we don't want to have this PR delayed just because of us~